crates/anvil/src/eth/api.rs

```diff
@@ -91,7 +91,7 @@ pub struct EthApi {
     pool: Arc<Pool>,
     /// Holds all blockchain related data
     /// In-Memory only for now
-    pub(super) backend: Arc<backend::mem::Backend>,
+    pub backend: Arc<backend::mem::Backend>,
```

Comment:
let's do pub(crate)

Reply:
The integration test crate requires access to the backend fns so pub(crate) won't suffice unless I move the test into the eth crate